nixos/modules/services/web-apps/mastodon.nix

```
@@ -100,6 +102,37 @@ let
     eval -- "\$@"
   '';
 
+  sidekiqUnits = lib.attrsets.mapAttrs' (name: processCfg:
+    lib.nameValuePair "mastodon-sidekiq-${name}" (let
+      jobClassArgs = lib.concatStringsSep " " (builtins.map (c: "-q ${c}") processCfg.jobClasses);
+      jobClassLabel = lib.concatStringsSep " " ([""] ++ processCfg.jobClasses);
```

```suggestion
      jobClassLabel = toString processCfg.jobClasses;
```

This should do the same job as it inserts spaces inbetween the entries.

---

good to know, thanks!